plugins/org.jkiss.dbeaver.data.office/src/org/jkiss/dbeaver/data/office/export/WorksheetUtils.java

```diff
@@ -31,7 +31,7 @@ public class WorksheetUtils {
 
     private static final Log log = Log.getLog(WorksheetUtils.class);
 
-    public static final String DEFAULT_SHEET_NAME = "Sheet";
+    public static final String DEFAULT_SHEET_NAME = "Custom query";
```

Comment:
Well, this sounds quite specific. Why a query, and why is it custom? Why don't we want to use `Sheet` instead?

Reply:
I'd prefer CustomQuery (no spaces).
And please fix formatting